includes/definitions/dasan-nos.yaml

```diff
@@ -0,0 +1,10 @@
+os: dasan-nos
+text: 'Dasan NOS'
+type: network
+icon: dasan
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+discovery:
+    method: sysObjectId
+    sysObjectId:
+        - .1.3.6.1.4.1.6296.1.2.5.12
```

Comment:
How would you do something had an array of sysObjectId to check AND an array of sysDescr?

What about if you want or instead of and? That's the reason I've put a method value in.